src/language/providers/closeElementSlash.ts

```diff
@@ -28,7 +28,7 @@ export function getCloseElementSlashProvider() {
   return vscode.languages.registerCompletionItemProvider(
     'dfdl',
     {
-      provideCompletionItems(
+      async provideCompletionItems(
```

Comment:
This was needed, along with adding `await` to line 54, for VSCode versions > 1.74.3 to avoid the auto delete of `>`.